### ISSUE TITLE
HA hangs at checking yaml

### ISSUE THREAD

**Issue by gurkburk76** [Anashost/WeatherXM-Home-Assistant#4]
As written in topic, it jsut hangs and stops me from restarting ha, not sure if it also would be prefered to use the "new" template format that was introduced a while ago.

**Comment by Anashost** [comment]
I just checked the code, im not having this issue. Make sure its not something else in your sensors.yaml that causing issues or conflicts.

I must say tho that im no longer using sensors.yaml method, instead im doing the following:

- In configurations.yaml i added: `sensor: !include_dir_merge_list sensors/`

- removed `sensor: !include sensors.yaml`

- Then created a folder called `sensors` in the main directory.

Inside this folder i create files with `.yaml` for my different templates, like `weatherXM.yaml` where i put yaml related for weatherXM only, to keeps things organized and avoid conflicts.

I hope this helps.

**Comment by gurkburk76** [comment]
Alright thanks, i'll look into that, sounds like a sane thing to do, break out stuff. :)

Oh, another thing, how often does it pull values from the API? is this something i could set or change ?

**Comment by Anashost** [comment]
>Oh, another thing, how often does it pull values from the API? is this something i could set or change ?

Everything 5 minutes. You can change this (in seconds), but i would not make it less that 1 minute to avoid getting blocked by the api. I'm not sure tho if they even care as of now.

`scan_interval: 300`

**Comment by gurkburk76** [comment]
(Reopening this as i don't want to create yet another issue.)
So i guess that you can't get *all* the values via the API as you can in the app, i'm mainly thinking forecast for X days and X hours and so on.
I've been fiddling with a few cards in HA like platinum-weather-card and i'm having a hard time populating above fields so i guess it can't be done right?

Is there any specific HA card you would recoment to get the most of out the info from this weatherstation?

**Comment by gurkburk76** [comment]
found this that seems to pull more data, but it's bugged so it does not refresh unless you reload the integration :\
https://community.home-assistant.io/t/weatherxm-integration-get-free-data-and-forecast-from-weather-stations-around-the-world/737574

**Comment by gurkburk76** [comment]
Looks like you can pull a whole lot more from the API if you know what you're doing (but i don't) :)
